files/image_config/caclmgrd/caclmgrd

```diff
@@ -147,6 +147,9 @@ class ControlPlaneAclManager(object):
         iptables_cmds.append("ip6tables -F")
         iptables_cmds.append("ip6tables -X")
 
+        # Add iptables command to allow all traffic from localhost
+        iptables_cmds.append("iptables -A INPUT -s 127.0.0.1 -j ACCEPT")
```

Comment:
What if someone sends you a packet with 127.0.0.1 dst address?
I'd better use something like this:
```
sudo iptables -A INPUT -i lo -j ACCEPT
sudo iptables -A OUTPUT -o lo -j ACCEPT
```

Reply:
Done. Agreed that an interface-based approach is more secure than an IP-based approach here.

Reply:
We have assigned 2 ipv4 address to lo:
1. 127.0.0.1
2. loopback address
I thought this method will allow more than we expected.

Reply:
Good point. Changed: Now accepting all incoming traffic on lo iff it has the localhost source IP.